pom.xml

```diff
@@ -60,5 +60,27 @@
             <artifactId>jackson-databind</artifactId>
             <version>2.9.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```

Comment:
What are you using lombok for?

Reply:
it was used for prev. version, will be removed with deprecated code.